Documentation/internals/security-identities.rst

```diff
@@ -4,6 +4,8 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. _security_identities:
+
```

Comment:
No need for this reference any more.

```suggestion
```

Reply:
@squeed This ref is still unused, I think it should be removed?

Reply:
I was just fixing that :)